sc/ccp-cell-cycle-genes.pickle.info

```diff
@@ -0,0 +1,24 @@
+{
+    "collection": "ArrayExpress",
+    "description": "A RNA-seq dataset based on Quartz-seq protocol. The data include 182 mouse embryonic stem cells (mESCs), which were FACS sorted into G1, S and G2M phases based on their Hoechst 33,342-stained cell area. The reduced data set includes more than 563 genes related to cell cycle based on Gene Ontology terms. The data have been deposited at ArrayExpress: <a href='https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3749/'> E-MTAB-3749.</a>",
```

Comment:
A RNA -> An RNA
more than 563 genes -> 563 genes ?? (or maybe 562 if one is the target?)